### DISCUSSION TITLE
World Mission Sunday

### DISCUSSION THREAD

**Discussion by ajmalton** [DivinumOfficium/divinum-officium#2648] [General]
Today being World Mission Sunday we (diocesan EF) had commemoratio Pro propaganda fidei sub una conclusione.

The day was instituted in 1926, but not at that time I think had it any liturgical place.

- Does anyone know when and for where the Commemoration was instituted?
- Should it be reflected in missa/ for some option?

**Comment by APMarcello3** [comment]
It should be reflected at least in 1960 rubrics for the 4th Sunday of October.

**Reply by ajmalton** [reply]
In occurrence with Christi Regis (which already has commemoration of the Sunday) what happens?

**Comment by gregordick** [comment]
The commemoration is made on the penultimate Sunday in October, so it can't occur with Christ the King.  With the 1960 rubrics this is always the "fourth Sunday of October" in the sense of that term used in the breviary, because in years in which there are only four Sundays that fall within the month of October, the third week is omitted.  However, that's not the case in earlier rubrics, where in years in which October starts on a Thursday, the penultimate Sunday in October is in the breviary's reckoning the third Sunday of October.

We have the "DU" designation for the last Sunday _in_ (as opposed to _of_) a month, for the sake of placing Christ the King; perhaps we need something similar for the penultimate Sunday in a month.

The 1960 general rubrics of the Missal say this on the subject of this commemoration:

> 453. On the next to last Sunday of October, or on another Sunday designated by the local ordinary as being "for the missions," in all Masses, the collect "For the Propagation of the Faith" is added to the collect of the Mass under a single conclusion, except on the days listed under nos. 1, 2, 3 and 8 in the table of precedence.

Those exceptions can't apply on the penultimate Sunday of October.

The commemoration was first prescribed in 1926 in a decree of the SRC that appeared in AAS 19 (1927), pp. 23-24.  Here it's an oratio imperata, meaning that the rules governing it are slightly different from those in the 1960 rubrics, and in particular it's not to be said under a single conclusion with the collect of the day (Additiones et variationes in rubricis Missalis, VI.8.4).

**Reply by ajmalton** [reply]
Excellent, thanks: answers all questions.
